exp-player/package.json

```diff
@@ -57,7 +57,8 @@
     "broccoli-merge-trees": "^1.1.1",
     "ember-cli-babel": "^5.1.5",
     "ember-cli-htmlbars": "0.7.4",
-    "ember-cp-validations": "2.9.2"
+    "ember-cp-validations": "2.9.2",
+    "ember-power-select": "0.10.11"
```

Comment:
This pins to a different version than the companion PR, which is mildly confusing.
https://github.com/CenterForOpenScience/isp/pull/89/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R50

Once that's resolved clear to merge.